packages/react-components/src/templates/TeamProfileHeader.tsx

```diff
@@ -98,7 +98,9 @@ const TeamProfileHeader: React.FC<TeamProfileHeaderProps> = ({
                   key={memberId}
                   css={[listItemStyles, { left: `-${i * 3}px` }]}
                 >
-                  <Anchor href={network({}).users({ userId: memberId }).$}>
```

Comment:
This typechecked????

Reply:
I suppose it's not limited to the params that actually exist. Unfortunate, but meh, the real problem I guess is that the `.user` part was missing, which can't really be avoided by typechecks. Once the `.user` part with the param is added, one will notice that there shouldn't be two routes that get the `userId` param